### ISSUE TITLE
More inocuous? python spew

### ISSUE THREAD

**Issue by zmerlynn** [GoogleCloudPlatform/kubernetes#4111]
This is showing up in the last few Jenkins logs, though it doesn't seem to stop them..

```
+++ Staging server tars to Google Storage: gs://kubernetes-staging-a978d/devel
Traceback (most recent call last):
  File "<string>", line 5, in <module>
TypeError: 'NoneType' object has no attribute '__getitem__'
```

**Comment by zmerlynn** [comment]
cc @jlowdermilk, but I doubt it

**Comment by jlowdermilk** [comment]
It looks like this has to be caused #4097, but it seems to be innocuous. I also can't get the error to repro, either locally or on the jenkins vm, but this should be resolved by addressing [#4100](https://github.com/GoogleCloudPlatform/kubernetes/issues/4100#issuecomment-72904573)

**Comment by jlowdermilk** [comment]
Error is gone now, I believe fixed by #4205.
